samples/interactive-tutorials/src/main/java/events/ImportUserEventsBigQuery.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START retail_import_user_events_from_big_query]
+
+/*
+ * Import user events into a catalog from GCS using Retail API
+ */
+
+package events;
+
+import com.google.cloud.retail.v2.BigQuerySource;
+import com.google.cloud.retail.v2.ImportMetadata;
+import com.google.cloud.retail.v2.ImportUserEventsRequest;
+import com.google.cloud.retail.v2.ImportUserEventsResponse;
+import com.google.cloud.retail.v2.UserEventInputConfig;
+import com.google.cloud.retail.v2.UserEventServiceClient;
+import com.google.longrunning.Operation;
+import com.google.longrunning.OperationsClient;
+import java.io.IOException;
+
+public class ImportUserEventsBigQuery {
+
+  public static void main(String[] args) throws IOException, InterruptedException {
+    String projectId = System.getenv("PROJECT_ID");
```

Comment:
`System.getenv...` is usually used in the integration test instead of the body of the sample. The convention should be like [this](https://github.com/googleapis/java-bigquery/blob/main/samples/snippets/src/main/java/com/example/bigquery/CreateDataset.java#L29-L30).

Reply:
it was requested to change the approach of storing the PROJECT_ID, we need to change our code so the PROJECT_ID will be read from the gcloud config or from the google.auth. We are working on this and will submit the saparate PR

Reply:
Fixed.